app/main.py

```diff
@@ -1,34 +1,73 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
```

Comment:
Provide types of tuple items

Reply:
The same in other places

Reply:
Example: tuple[str, str, int] - mean 1 parameter is str, second str, and third int